h2/src/main/org/h2/tools/CreateCluster.java

```diff
@@ -169,7 +190,6 @@ private void process(String urlSource, String urlTarget,
                 statSource.execute("SET EXCLUSIVE FALSE");
             }
         } finally {
-            FileUtils.delete(scriptFile);
             JdbcUtils.closeSilently(statSource);
```

Comment:
need to close the pipedreader here

Reply:
Thanks, all good suggestions, I'll make the changes and let you know. :)